### ISSUE TITLE
Shell.Current.FlyoutIsPresented = true still does not toggle the first time on Android

### ISSUE THREAD

**Issue by criss02-cs** [dotnet/maui#20549]
### Description

Toggle Shell.Current.FlyoutIsPresented doesn't open Flyout on first time. I have to open manually

### Steps to Reproduce

1. Clone the project
2. Run ObsidianUI.Examples on Android
3. Try to click Try out

### Link to public reproduction project repository

https://github.com/criss02-cs/ObsidianUI

### Version with bug

8.0.6

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Did you find any workaround?

```
if (Shell.Current.FlyoutIsPresented)
{
    Shell.Current.FlyoutIsPresented = false;
}
else
{
    Shell.Current.FlyoutBehavior = FlyoutBehavior.Locked;
    Shell.Current.FlyoutBehavior = FlyoutBehavior.Flyout;
    Shell.Current.FlyoutIsPresented = true;
}
```

For now I found this workaround, but it's not possible to do this everytime

**Comment by PureWeen** [comment]
Can you test with the latest nightly build? 
https://github.com/dotnet/maui/wiki/Nightly-Builds

**Comment by criss02-cs** [comment]
Hi, with 8.0.10-nightly.10055 version it works. When this verson will be available?

**Comment by PureWeen** [comment]
This should be available in 8.0.7 which went public two days ago
https://github.com/dotnet/maui/issues/20159
